vmdb/app/views/shared/dialogs/_dialog_provision.html.haml

```diff
@@ -15,7 +15,7 @@
       - wf.dialog.dialog_tabs.each do |tab|
         = render :partial => "shared/dialogs/dialog_tab", 
                  :locals => {:wf => wf, :tab => tab}
-  - unless @edit[:explorer]
+  - if @edit && !@edit[:explorer]
```

Comment:
@h-kataria So, we need to check `@edit.nil?` whenever we're not on a summary page?  Sounds painful.  Using hashes everywhere is probably not ideal but can't we initialize the hash in one place when we aren't on a summary page?

This is just curiosity... I don't know how bad this would be to fix in a different way.